Tweetinvi.Streams/AccountActivityStream.cs

```diff
@@ -228,18 +228,20 @@ private void TryRaiseUserEvent(string eventName, JObject jsonObjectEvent)
         private void TryRaiseMessageEvent(string eventName, JObject jsonObjectEvent)
         {
             var messageEventDTOs = jsonObjectEvent[eventName].ToObject<EventDTO[]>();
-            var apps = jsonObjectEvent["apps"].ToObject<Dictionary<string, App>>();
+            var apps = jsonObjectEvent.GetValue("apps")?.ToObject<Dictionary<string, App>>();
+            var users = jsonObjectEvent.GetValue("users")?.ToObject<Dictionary<string, MessageUser>>().Select((x) => x.Value).ToList();
 
             messageEventDTOs.ForEach(messageEventDTO =>
             {
                 App app = null;
 
                 if (messageEventDTO.MessageCreate.SourceAppId != null)
                 {
-                    apps.TryGetValue(messageEventDTO.MessageCreate.SourceAppId.ToString(), out app);
+                    apps?.TryGetValue(messageEventDTO.MessageCreate.SourceAppId.ToString(), out app);
                 }
 
                 var message = _messageFactory.GenerateMessageFromEventDTO(messageEventDTO, app);
```

Comment:
As it is a factory, it should take the `users` as a new parameter instead of assigning the value.